plugin/tokudb-backup-plugin/tokudb_backup.cc

```diff
@@ -42,19 +42,15 @@ static MYSQL_SYSVAR_STR(version, tokudb_backup_version,
     "version of the tokutek backup library",
     NULL, NULL, NULL);
 
-static MYSQL_THDVAR_ULONG(last_error,
+static MYSQL_THDVAR_INT(last_error,
     PLUGIN_VAR_THDLOCAL,
     "error from the last backup. 0 is success",
-    NULL, NULL, 0, 0, ~0ULL, 1);
-
-static void tokudb_backup_update_last_error_str(THD* thd,
-                                                struct st_mysql_sys_var* var,
-                                                void* var_ptr, const void* save);
+    NULL, NULL, 0, 0, 0, 1);
```

Comment:
This was a Tokutek thing. This 'variable' is never, ever meant to be set by a user, only by the plugin, which can set it to any value regardless of the SYSVAR limits. It has no meaning whatsoever if a user sets the value. These should have been done as session status, not system variables.